hrjobcontract/api/v3/HRJobContract.php

```diff
@@ -92,14 +92,31 @@ function civicrm_api3_h_r_job_contract_deletecontract($params) {
 }
 
 /**
- * HRJobContract.deletecontract API
+ * HRJobContract.deletecontractpermanently API
  *
  * @param array $params
  * @return array API result descriptor
  * @throws API_Exception
  */
 function civicrm_api3_h_r_job_contract_deletecontractpermanently($params) {
-  return _civicrm_hrjobcontract_api3_deletecontractpermanently($params);
+  if (empty($params['jobcontract_id'])) {
```

Comment:
It would be nice to have api tests to check that you'll get these errors if the required param is not present